packages/shadergradient/src/UI/Inputs/ColorInput/ColorInput.tsx

```diff
@@ -41,78 +41,80 @@ export const ColorInput = React.forwardRef<HTMLInputElement, ColorInputPropsT>(
         )}
         <div className='flex items-center gap-2 w-full relative'>
           <div
-            className='w-full h-input rounded'
+            className='w-full h-input rounded relative'
             style={{ background: sharedValue }}
             onClick={() => {
               setToggle(!toggle)
             }}
-          ></div>
-
-          {/* color control */}
-          <div
-            ref={tabRef}
-            style={{
-              width: 'fit-content',
-              height: 290,
-              position: 'absolute',
-              bottom: 0,
-              left: -10,
-              zIndex: 100,
-              display: toggle === true ? 'block' : 'none',
-            }}
           >
             <div
+              ref={tabRef}
               style={{
-                position: 'fixed',
-                display: 'flex',
                 width: 'fit-content',
-                height: 'fit-content',
-                background: 'white',
-                padding: 16,
-                flexDirection: 'column',
-                justifyContent: 'center',
-                alignItems: 'center',
-                gap: 16,
-                borderRadius: 5,
-                filter: 'drop-shadow(0px 0px 10px rgba(0,0,0,0.37))',
+                height: 290,
```

Comment:
@stone-skipper how about increasing the `bottom` value instead of making fixed `height` (290) that is hard to guess its purpose